ncm-nfs/src/main/pan/components/nfs/schema.pan

```diff
@@ -15,7 +15,7 @@ type structure_nfs_exports = {
 type structure_nfs_mounts = {
     'device'     : string
     'mountpoint' : string
-    'fstype'     : string with match(SELF, '^nfs|none$')
+    'fstype'     : string with match(SELF, '^nfs|panfs|none$')
```

Comment:
i didn't mean to imply that you introduced the support for nfsbanana, but might as well fix it :smile: